pkg/projectfile/projectfile.go

```diff
@@ -923,14 +923,6 @@ type CreateParams struct {
 	Cache      string
 }
 
-// TestOnlyCreateWithProjectURL a new activestate.yaml with default content
-func TestOnlyCreateWithProjectURL(projectURL, path string) (*Project, error) {
-	return createCustom(&CreateParams{
-		ProjectURL: projectURL,
-		Directory:  path,
-	}, language.Python3)
-}
```

Comment:
Apologies for sneaking this into my PR, it's not directly related to the story but I ran into it for one of the failing tests and wanted to do a quick cleanup. If it distracts from the review I'll open a separate PR.

This code shouldn't have been inside a non-test file.

Reply:
That's fine. I'm in favor of fixing things as you find them. It's faster and you stay in the zone.